static/js/factories/learning_resources.js

```diff
@@ -24,7 +24,7 @@ import type {
 const incrCourse = incrementer()
 const courseId: any = incrementer()
 
-const dateFormat = "YYYY-MM-DD[T]HH:mm:ss[Z]"
+export const dateFormat = "YYYY-MM-DD[T]HH:mm:ss[Z]"
```

Comment:
I think we should probably move this constant to a constants file and rename it so its `ALL_CAPS` like we typically do for constants in the application code - first because we don't want any imports in the application code from the factory module, and second because when it's used in the `lib` file below `dateFormat` looks like it's a function, not a constant

Reply:
✔️